src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3132,4 +3132,7 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Sort_Imports_or_usings" xml:space="preserve">
     <value>Sort Imports or usings</value>
   </data>
+  <data name="Insert_an_if_statement" xml:space="preserve">
+    <value>Insert an if statement</value>
```

Comment:
I saw you mentioned this bit in a test. I forget, are we guaranteed to have a singular locale at CI time or is the test resilient?

Reply:
not sure what you mean here

Reply:
Not sure if we occasionally use CI's that are different locales than english. If we do then the FeatureResource acquisition of `Insert_an_if_statement` will be different depending on the lang